src/makeRobot.js

```diff
@@ -38,7 +38,57 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, `
+      + `chip version: ${this.version}, wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
+    },
+
+    goForward(coord = 1) {
```

Comment:
```suggestion
    goForward(step = 1) {
```
It would be more meaningful. The argument of the method `goForward` is a number on which should coords change. So it's actually steps that robot will make